From kristen.c.accardi@intel.com Fri Sep 29 10:30:59 2006
Date: Fri, 29 Sep 2006 10:30:27 -0700
From: Kristen Carlson Accardi <kristen.c.accardi@intel.com>
To: Greg Kroah-Hartman <greg@kroah.com>
Cc: pcihpd-discuss@lists.sourceforge.net, linux-kernel@vger.kernel.org
Subject: [patch] change pci hotplug subsystem maintainer to Kristen
Message-Id: <20060929103027.84bc7aab.kristen.c.accardi@intel.com>

Here's a patch adding me to the maintainers file for the pci
hotplug subsystem, as we discussed.

Signed-off-by: Kristen Carlson Accardi <kristen.c.accardi@intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 MAINTAINERS                            |    4 ++--
 drivers/pci/hotplug/pci_hotplug.h      |    2 +-
 drivers/pci/hotplug/pci_hotplug_core.c |    4 +---
 3 files changed, 4 insertions(+), 6 deletions(-)

--- gregkh-2.6.orig/MAINTAINERS
+++ gregkh-2.6/MAINTAINERS
@@ -2268,8 +2268,8 @@ T:	quilt kernel.org/pub/linux/kernel/peo
 S:	Supported
 
 PCI HOTPLUG CORE
-P:	Greg Kroah-Hartman
-M:	gregkh@suse.de
+P: 	Kristen Carlson Accardi
+M:	kristen.c.accardi@intel.com
 S:	Supported
 
 PCI HOTPLUG COMPAQ DRIVER
--- gregkh-2.6.orig/drivers/pci/hotplug/pci_hotplug.h
+++ gregkh-2.6/drivers/pci/hotplug/pci_hotplug.h
@@ -22,7 +22,7 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  *
- * Send feedback to <greg@kroah.com>
+ * Send feedback to <kristen.c.accardi@intel.com>
  *
  */
 #ifndef _PCI_HOTPLUG_H
--- gregkh-2.6.orig/drivers/pci/hotplug/pci_hotplug_core.c
+++ gregkh-2.6/drivers/pci/hotplug/pci_hotplug_core.c
@@ -21,9 +21,7 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  *
- * Send feedback to <greg@kroah.com>
- *
- * Filesystem portion based on work done by Pat Mochel on ddfs/driverfs
+ * Send feedback to <kristen.c.accardi@intel.com>
  *
  */
 
